sgl-kernel/csrc/cpu/gemm.cpp

```diff
@@ -439,7 +439,7 @@ at::Tensor weight_packed_linear(at::Tensor& mat1, at::Tensor& mat2,
   const float* bias_data = nullptr;
   if (has_bias) {
     CHECK_EQ(bias.value().size(0), N);
-    bias_data = bias.value().data_ptr<float>();
+    bias_data = bias.value().to(at::kFloat).data_ptr<float>();
```

Comment:
change `bias` from bfloat16 to float when loading the model would be a better idea as this won't hurt the performance.

Reply:
I move the code to `_process_weight_after_loading()`. Let me know if it addresses the problem.